update.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# this script updates your server to the latest version available
+
+# change to server directory
+cd dirname/minecraft/
+
+# execute stop script
+./stop.sh
+
+# grep latest verion from papermc.io
+Version=$(wget -q -O - https://papermc.io/api/v2/projects/paper | rev | cut -d, -f1 | rev)
+Version="${Version:1}"
+Version="${Version::-3}"
+echo "latest server version seems to be $Version"
+echo "trying to update to version $Version ..."
```

Comment:
Maybe prompt the user if they want to update

Reply:
Well, personally I prefer user input, but some people want a "it just works" solution. I don't know what people prefer more. Of course I can build in a confirmation prompt.